lnd_test.go

```diff
@@ -7657,20 +7657,16 @@ func testGraphTopologyNotifications(net *lntest.NetworkHarness, t *harnessTest)
 	// The channel opening above should have triggered a few notifications
 	// sent to the notification client. We'll expect two channel updates,
 	// and two node announcements.
-	const numExpectedUpdates = 4
-	for i := 0; i < numExpectedUpdates; i++ {
+	var numChannelUpds int
```

Comment:
Reminder that this diff should go away after rebase, since it was included in: da56aa3528ae95398060de99101c4b7b5d2ae5c3